ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -397,6 +400,955 @@ objects:
+                  "source": "data:text/plain;base64,${PROXY_KEY}"
+                },
+                "mode": 420
+              },
+              {
+                "filesystem": "root",
+                "path": "/tmp/squid/ca-chain.pem",
+                "user": {
+                  "name": "root"
+                },
+                "contents": {
+                  "source": "data:text/plain;base64,${CA_CHAIN}"
+                },
+                "mode": 420
+              },
+              {
+                "filesystem": "root",
+                "path": "/tmp/squid/squid.conf",
+                "user": {
+                  "name": "root"
+                },
+                "contents": {
+                  "source": "data:text/plain;base64,${SQUID_CONFIG}"
+                },
+                "mode": 420
+              },
+              {
+                "filesystem": "root",
+                "path": "/tmp/squid.sh",
+                "user": {
+                  "name": "root"
+                },
+                "contents": {
+                  "source": "data:text/plain;base64,${SQUID_SH}"
+                },
+                "mode": 420
+              },
+              {
+                "filesystem": "root",
+                "path": "/tmp/squid/proxy.sh",
+                "user": {
+                  "name": "root"
+                },
+                "contents": {
+                  "source": "data:text/plain;base64,${PROXY_SH}"
+                },
+                "mode": 420
+              },
+              {
+                "filesystem": "root",
+                "path": "/tmp/squid/passwd.sh",
+                "user": {
+                  "name": "root"
+                },
+                "contents": {
+                  "source": "data:text/plain;base64,${KEY_PASSWORD}"
+                },
+                "mode": 493
+              }
+            ]
+          },
+          "systemd": {
+            "units": [
+              {
+                "contents": "[Service]\n\nExecStart=bash /tmp/squid.sh\n\n[Install]\nWantedBy=multi-user.target\n",
+                "enabled": true,
+                "name": "squid.service"
+              },
+              {
+                "dropins": [
+                  {
+                    "contents": "[Service]\nExecStart=\nExecStart=/usr/lib/systemd/systemd-journal-gatewayd \\\n  --key=/opt/openshift/tls/journal-gatewayd.key \\\n  --cert=/opt/openshift/tls/journal-gatewayd.crt \\\n  --trust=/opt/openshift/tls/root-ca.crt\n",
+                    "name": "certs.conf"
+                  }
+                ],
+                "name": "systemd-journal-gatewayd.service"
+              },
+              {
+                "enabled": true,
+                "name": "systemd-journal-gatewayd.socket"
+              }
+            ]
+          }
+        }
+        EOF
+        }
+
+        function generateBlackholeTemplate() {
+          cat > /tmp/01_blackhole_vpc.yaml << EOF
```

Comment:
@wking is this a template that we can host [here](https://github.com/openshift/installer/tree/master/upi/aws/cloudformation) so users have a reference config ?